README.md

```diff
@@ -222,6 +222,31 @@ Autoload can be customised using the following options:
   })
   ```
 
+- `routeParams` (optional) - Folders prefixed with `_` will be turned into route parameters.
```

Comment:
There is also `routeParamPattern` to document

Reply:
Is there an example of a __warning: this could break things__ note elsewhere in the Fastify documentation?

If we expose this as a configuration option, I think it's important to point out that if this is set improperly it will cause all sorts of headaches (e.g. forgetting `^` and suddenly all of your routes are parametric)

Reply:
Stating it like you said it's ok.

I'm also ok in not making it configurable - however it's configurable in the code.

Reply:
Good call, I think it's better to move it out of the configuration scope for simplicity's sake. Prefixing with `_` is a good convention, and I think anything more complex is probably better served in application code instead of the filesystem.